README.md

```diff
@@ -91,9 +95,24 @@ To find out more about all the different ways to contribute to the BIDS Starter
 If you're here during summer 2018 :icecream::palm_tree:, you should reach out to our lovely [Google Summer of Code][gsoc] student [Patrick Park][patrick-github]. It would really help his project along if you said hello and passed along any feedback you have :purple_heart:. Please don't be shy, **the newer you are the more valuable your feedback is** :thumbsup:
 
 ## Acknowledgements
-![incf](https://www.eudat.eu/sites/default/files/styles/medium/public/logo/INCF_0.png?itok=uRT54XCM)
-![gsoc](https://upload.wikimedia.org/wikipedia/commons/thumb/1/1e/GSoC.png/220px-GSoC.png)
 
+### Sponsor Organizations
+
```

Comment:
@chrisfilo - at the moment we're only acknowledging INCF and GSOC ....should there be (Stanford Center for Reproducible Neuroscience)[http://reproducibility.stanford.edu) or anything/one else?

Reply:
(Laura and John Arnold Foundation)[http://www.arnoldfoundation.org/laura-and-john-arnold-foundation-announces-3-8-million-grant-to-stanford-university-to-improve-the-quality-of-neuroscience-research/], (National Institute for Mental Health)[http://grantome.com/grant/NIH/R24-MH114705-01], GSoC, INCF.

Reply:
This should really go to https://github.com/INCF/BIDS

Reply:
👍 Thanks @chrisfilo - @Park-Patrick, please add these to the README 😄 

I don't think you link to pages from the images at the moment, but I like the idea of doing the links that Chris has added. You could link to the GSOC and INCF home pages too.

Reply:
@chrisfilo - sorry - I think our comments crossed.

Do you mean this section of the README should also be in the main BIDS repo? If yes, I think that's a great idea!

Reply:
yes

Reply:
Added the two other organizations and changed the linked the images to their respective websites as well ✅